### PR TITLE
Change log to node's URL

### PR DESCRIPTION
**Pull request by nisdas**
- [x] Change from multiaddr to enode.
- [x] Use libp2p's secpk2561 keys

### CODE REVIEW

File: tools/bootnode/bootnode.go [hunk 1]
```diff
@@ -88,10 +84,11 @@ func extractPrivateKey() *ecdsa.PrivateKey {
 		}
 
 	} else {
-		privKey, err = ecdsa.GenerateKey(ECDSACurve, rand.Reader)
+		privInterfaceKey, _, err := crypto.GenerateSecp256k1Key(rand.Reader)
```
Comment (shayzluf) [inline tools/bootnode/bootnode.go]: do x509.ParseECPrivateKey(b) supports Secp256k1Key?

Comment (nisdas) [reply tools/bootnode/bootnode.go]: this does, I can add a test for this code path

Comment (shayzluf) [reply tools/bootnode/bootnode.go]: great

Comment (nisdas) [reply tools/bootnode/bootnode.go]: Thanks for catching that! turns out it doesn't parse correctly, so I unmarshalled it using another one from the libp2p crypto library.

**Pull request merged**